scripts/kani-perf.sh

```diff
@@ -8,8 +8,8 @@ set -o nounset
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 KANI_DIR=$SCRIPT_DIR/..
 
-# Build Kani using release mode.
-cargo build --release
+# Build Kani
+cargo build
```

Comment:
Shouldn't we use release though?

Reply:
Yes, we should, but our `kani` and `cargo-kani` dev scripts currently do not support having two binaries in the build directory. 

I could add a `cargo clean` before the `cargo build --release` line to make sure other binaries are removed, but this might be unexpected by the script user.

Another alternative is to build Kani with `--release` in the GH action:
https://github.com/model-checking/kani/blob/61c8d1180fc49cce8d730f6ebdfb6d6198fd610b/.github/workflows/kani.yml#L67

But this may fail locally if someone tries to run the `kani-perf.sh` script directly.

Thoughts?

Reply:
Maybe set the CI to build with release for now. I might fix this issue in my next change.

Reply:
Sounds good. Done.